### PR TITLE
PT: The Complete Title Screen UI file 

### PR DESCRIPTION
**Pull request by BadHowst**
The Complete Title Screen UI file  and readme file for the new engine

### CODE REVIEW

File: Interface/ui_title_text.csv [hunk 1]
```diff
@@ -99,60 +99,60 @@ Language_Used#0,"""-"""
...
-Notice#0,"""ワンタイムパスワードを<yellow>半角<c>で入力してください。<br><br>ワンタイムパスワードは発行されてから<br><yellow>約1分間<c>で使えなくなります。<br>その場合は、あらためて「再発行」をお願いします。<br><br>また、<red>6回連続<c>で間違えますと、ロック状態となります。<br>ロック状態の解除に関する情報は<br>プレイヤーズサイトにてご確認ください。"""
-Send#1,"""決定"""
-Cancel#1,"""戻る"""
-WebLinkButton#0,"""認証ができないなどのトラブルはこちら"""
-FormName#2,"""プレイヤーIDネーム入力"""
-Notice#1,"""プレイヤーIDネームとは<br>『ファンタシースターオンライン2』の中で<br>各プレイヤーを判別するための名前のことです。<br><br>設定したプレイヤーIDネームは、<br><yellow>他のユーザーに公開されます。<c><br><br>他のプレイヤーと重複するプレイヤーIDネームは<br>設定できませんのでご注意ください。<br><br>半角文字だけでなく、<yellow>全角文字も使用できます。<c>"""
-Change_FormName#0,"""プレイヤーIDネーム変更"""
+FormName#0,"""Insira Chave para Autenticar"""
+Message#0,"""Voce precisa inserir uma [Chave para Autenticar do E-mail]<br>para jogar. Porfavor insira [Chave para Autenticar do E-mail]<br>que foi enviada para o seu e-mail<br>registrado no seu SEGA ID."""
+Send#0,"""Confirmar""
```
Comment (alama) [inline Interface/ui_title_text.csv]: There is a quote missing here in file "Interface/ui_title_text.csv", line 125, ID Send#0

**Review by alama — CHANGES_REQUESTED** [review]
Broken CSV file

File: Interface/ui_title_text.csv (hunk 1 above)
Comment (BadHowst) [reply Interface/ui_title_text.csv]: already fixed

File: Interface/ui_title_text.csv [hunk 2]
```diff
@@ -54,41 +54,41 @@ UseAgreement_Message_ps4_break2#0,"""第11条（コミュニティ機能等）<b
...
-Recommand_Start#0,"""ゲームを開始する"""
-Recommand_Back#0,"""戻る"""
-LoginCloud#0,"""ゲームスタート"""
-LoginIdSelect_es#0,"""『PSO2es』との連動"""
-LoginIdSelect_es_Title#0,"""『ＰＳＯ２ｅｓ』とキャラクターを連動するには"""
-LoginIdSelect_es_Message#0,"""『ＰＳＯ２ｅｓ』のキャラクターで『ＰＳＯ２』をプレイするには<br>『ＰＳＯ２ｅｓ』から連動処理を行う必要があります。<br><br>『ＰＳＯ２ｅｓ』ロビー画面の「ＰＳＯ２連動」メニューから<br>「今すぐ連動する」ボタンをタッチし<br>メニューの説明にしたがって、連動を完了してから<br>『ＰＳＯ２』にログインしてください。<br><br>\u3000<&URL pso2.jp_es_players_guide_beginners>"""
+FirstPlay_Message#0,"""Bemvindo ao Phantasy Star Online 2!<br><br>Neste jogo voce pode selecionar uma<green>\""Nave""<c> e registre seu perosnagem.<br><br><yellaow>■Para Iniciantes:<c><br>Por favor selecione\""Iniciando o jogo pela primeira vez\"".<br><br><yellow>■Para aqueles que foram convidados por amigos:<c><br>Por favor selecione a Nave em que seu amigo esta.<br><br><yellow>■para aqueles que ja jogam:<c><br>Selecione a Nave em que ja joga."""
```
Comment (alama) [inline Interface/ui_title_text.csv]: The text in "FirstPlay_Message#0" is broken
`\""Nave""<c>`
you need to change this to:
`\""Nave\""<c>`

**Review by alama — CHANGES_REQUESTED** [review]
Broken Quote

File: Interface/ui_title_text.csv (hunk 2 above)
Comment (BadHowst) [reply Interface/ui_title_text.csv]: Fixed

Comment (alama) [reply Interface/ui_title_text.csv]: Sorry, this is not fixed...

Comment (BadHowst) [reply Interface/ui_title_text.csv]: yeh i noticed my mistake after i did it again, this line should me fine now

Comment (BadHowst) [reply Interface/ui_title_text.csv]: all the changes needed are done 
ready for posting

**Comment by BadHowst** [comment]
All the changes are done, there is a slight issue right now with the ReadMe file but nothing to worrie about, the file is ready to be published

**Pull request merged**